modules/distr-tracing-install-jaeger-operator.adoc

```diff
@@ -5,40 +5,35 @@ This module included in the following assemblies:
 
 :_mod-docs-content-type: PROCEDURE
 [id="distr-tracing-jaeger-operator-install_{context}"]
-= Installing the {JaegerName} Operator
+= Installing the {JaegerOperator} Operator
 
-To install {JaegerName}, you use the link:https://operatorhub.io/[OperatorHub] to install the {JaegerName} Operator.
+You can install the {JaegerOperator} Operator through the link:https://operatorhub.io/[*OperatorHub*].
 
 By default, the Operator is installed in the `openshift-operators` project.
 
 .Prerequisites
 * You have access to the {product-title} web console.
 * You have access to the cluster as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
-* If you require persistent storage, you must also install the OpenShift Elasticsearch Operator before installing the {JaegerName} Operator.
-
-[WARNING]
-====
-Do not install Community versions of the Operators. Community Operators are not supported.
-====
+* If you require persistent storage, you must also install the OpenShift Elasticsearch Operator before installing the {JaegerOperator} Operator.
 
 .Procedure
 
 . Log in to the {product-title} web console as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
 
 . Navigate to *Operators* -> *OperatorHub*.
 
-. Type *distributed tracing platform* into the filter to locate the {JaegerName} Operator.
+. Search for the {JaegerOperator} Operator by entering *distributed tracing platform* in the search field.
 
-. Click the *{JaegerName} Operator* provided by Red Hat to display information about the Operator.
+. Select the *{JaegerOperator}* Operator, which is *provided by Red Hat*, to display information about the Operator.
 
 . Click *Install*.
 
-. On the *Install Operator* page, select the *stable* Update Channel. This automatically updates your Operator as new versions are released.
+. On the *Install Operator* page, select the *stable* for the *Update channel*. This will automatically update the Operator when new versions are released.
```

Comment:
```suggestion
. On the *Install Operator* page, select *stable* for *Update channel*. This setting automatically updates the Operator when new versions are released.
```

Reply:
See https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs look for "tense" - the future tense in cause-and-effect sentences is discouraged.

Reply:
Thank you. I'm generally careful to avoid the future tense, but I didn't know about this IBM Style rule. I've thought about this sentence some more, and here is my end result:
```suggestion
. For the *Update channel* on the *Install Operator* page, select *stable* to automatically update the Operator when new versions are released.
```